### PR TITLE
feat: Update release note ALA-845

### PR DESCRIPTION
**Pull request by nicklem**
Updates the Coverage release note to complete the list of endpoint/UI migrations

### :eyes: Live preview
https://ala-845-p2-doc-update-release-note--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks/

### TODO:
- [x] Add release dates
- [x] Potentially flag as migrated also the Quality PR and Commit pages
- [x] Review links

### CODE REVIEW

**Review by machadoit — APPROVED** [review]
Nice!

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md [hunk 1]
```diff
@@ -172,9 +170,25 @@ On December 7th 2023 we set the new Coverage engine to post coverage summaries t
     end="<!--accept-permission-end-->"
 %}
 
+### Quality dashboard, Coverage dashboard, Quality pull request list page, and Coverage pull request list page {: id="migrated-pages" }
```
Comment (machadoit) [inline docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:173]: Actually wait, I am not sure if all the info is from new engine, I doubt it, I think we should not mention those

**Review by machadoit — CHANGES_REQUESTED** [review]

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md [hunk 2]
```diff
@@ -36,45 +36,43 @@ Please refer to the table below for the updated status of the transition process
...
       <td>Planned</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>
-        <a href="#diff-tabs">Coverage pull request page</a>
-      </td>
-      <td>Live (partially migrated)</td>
+      <td><a href="#diff-tabs">Coverage pull request page</a></td>
+      <td>In progress</td>
```
Comment (alerizzo) [inline docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:65]: This one should be Live, right?

Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:65]: Aren't we getting data from the new engine only for the Diff tab? 🤔

**Review by machadoit — APPROVED** [review]
LGTM!

**Review by machadoit — CHANGES_REQUESTED** [review]
Actually I missed that we were changing those things to live. Dashboards should remain on the planned

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md (hunk 1 above)
Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md]: Updated!

**Review by machadoit — APPROVED** [review]
Thank you! 🙏

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md (hunk 2 above)
Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md]: Updated.

**Pull request merged**